### PR TITLE
add version open orchestra in globals twig by config file

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

**Comment by SOLenG** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1183

File: BaseBundle/DependencyInjection/Compiler/TwigGlobalsCompilerPass.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+     *
+     * @api
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if ($container->hasDefinition('twig')) {
+            $twig = $container->getDefinition('twig');
+
+            if($container->hasParameter('open_orchestra')) {
+                //$twig->addMethodCall('addGlobal', array('open_orchestra', array('version' => '1.1.0')));
```
Comment (aloupfor) [inline BaseBundle/DependencyInjection/Compiler/TwigGlobalsCompilerPass.php]: commented line?

**Comment by nicolasThal** [comment]
As we said, the version number should be a constant in the BaseBundle and not a container parameter.

See https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/Kernel.php .

I would suggest to put the const  in the `OpenOrchestraBaseBundle` file

**Comment by SOLenG** [comment]
@nicolasThal method fix

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**